pkg/window/keyed/keyed.go

```diff
@@ -19,10 +19,9 @@ limitations under the License.
 package keyed
 
 import (
-	"sync"
-
 	"github.com/numaproj/numaflow/pkg/pbq/partition"
 	"github.com/numaproj/numaflow/pkg/window"
+	"sync"
```

Comment:
can you enable this setting? 
<img width="338" alt="Screen Shot 2022-11-11 at 9 05 15 AM" src="https://user-images.githubusercontent.com/191430/201392309-74b8b058-783e-44dc-9890-0f12d0c1a7f4.png">

Reply:
done.